apps/api/src/users/dto/create-user.dto.ts

```diff
@@ -82,4 +83,7 @@ export class CreateUserDto {
   @IsNotEmpty()
   @IsPhoneNumber()
   readonly phoneNumber: string;
+
+  @IsBoolean()
+  readonly isActive: boolean;
```

Comment:
remove this since we can define this in service as default to true

Reply:
removed